packages/perseus/src/components/math-input.tsx

```diff
@@ -399,6 +399,21 @@ const MathInputIcon = ({hovered, focused, active}) => {
     );
 };
 
+const debounce = <T extends unknown[]>(
```

Comment:
I imagine we're defining our own debounce function because we're trying to reduce our usage of underscore and because this will provide better types.

I'd advocate moving this out of `math-input` and into `perseus/src/util.ts` or one of the files in `perseus/src/util`, but I'm not going to lose sleep if we keep it here for now.

Reply:
It didn't even occur to me to try to use underscore, to be honest. That implementation is probably fine! Switching now.

Reply:
jk, underscore's debounce causes a million test failures.